server/src/server.ts

```diff
@@ -307,7 +311,7 @@ function addClassPropertiesMethodsParentClassesAndTraits(toReturn: CompletionIte
 
     classNode.methods.forEach((subNode) => {
         var accessModifier = "method " + buildAccessModifier(subNode.accessModifier);
-        var insertText = subNode.name + "()";
+        var insertText = subNode.name + "(" + stringifyMethodParams(subNode) +")";
```

Comment:
As above - can you remove this from this pull request please? We'll discuss this separately in #44